docs/DeveloperGuide.md

```diff
@@ -491,7 +491,70 @@ shows you the welcome screen for the financial planner app
    1. Type `exit` into the terminal. 
    2. Expected: the financial planner will exit with a goodbye message.
 Under the data newly created data directory, a watchlist.json and a data.txt file will be created
-3. WatchList Feature
+
+### View Balance
+
+Test case: `balance`
+
+Expected: Balance is displayed. Details of balance are shown in the status message.
+
+### Budget Feature
+
+1. Setting a monthly budget
+
+Test case: `budget set /b 100`
+
+Expected: A monthly budget of 100 is set. Details of the budget are shown in the status message.
+
+Test case: `budget set /b`
+
+Expected: No budget is set. Error details shown in status message.
+
+Other incorrect set budget commands to try: `budget set`, `budget set /b x`, `...` (where x is negative)
```

Comment:
what is ...

Reply:
other incorrect commands for tested to come up with